requirements-dev.txt.in

```diff
@@ -1,7 +1,7 @@
-# Be sure to run "make requirements-dev.txt" after editing this file.
+# Be sure to run "make requirements-dev.txt" after editing this file and commit the resulting changes to both files.
 flake8
 mypy >= 0.521
-domovoi
+domovoi >= 1.4.2
```

Comment:
Given that the latest version in pypi is 1.4.2, I'm not sure what this accomplishes.  Is there a tool that compares what's installed against requirements.txt?